.github/workflows/release.yaml

```
@@ -9,7 +9,9 @@ on:
 env:
```

Would it make sense to have a "dry run" parameter that builds and signs the releases and verifies them, just so we can test the signing workflow?

```suggestion
      dry-run:
        description: "Build, sign, verify but do not release"
        required: false
        type: boolean
        default: true
env:
```